mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1159,6 +1159,22 @@ func ResourceContainerCluster() *schema.Resource {
 								},
 							},
 						},
+						"advanced_datapath": {
```

Comment:
Let's mirror the API name exactly, i.e. `advanced_datapath_observability_config`